contracts/BaseStrategy.sol

```diff
@@ -174,13 +191,6 @@ abstract contract BaseStrategy {
         return vault.governance();
     }
 
-    /*
-     * Provide an accurate expected value for the return this strategy
-     * would provide to the Vault the next time `report()` is called
-     * (since the last time it was called)
-     */
-    function expectedReturn() public virtual view returns (uint256);
```

Comment:
What's the rationale of removing `expectedReturn`?

Reply:
there were basically two `expectedReturn()`, one in the vault, and one in the strategy.
There was no real use for the strategy's one.

Reply:
Yeah, having the Vault maintain that statistic as "realized returns" seemed more appropriate.